src/D2L.CodeStyle.TestAnalyzers/NUnit/CategoryAnalyzer.cs

```diff
@@ -69,9 +70,16 @@ internal sealed class CategoryAnalyzer : DiagnosticAnalyzer {
 					return;
 				}
 
+				var syntaxReference = attribute.ApplicationSyntaxReference;
+
+				if( syntaxReference == null ) {
+					// ??

```

Comment:
I don't know if this is possible but also don't care much. I changed the behaviour to just silently continue if it happens